### PR TITLE
Fix: uploader falls back to regular upload API if chunked unsupported

### PR DESCRIPTION
**Pull request by DanDeMicco**
#466

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @DanDeMicco has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #519

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

**Comment by JustinHoldstock** [comment]
One quick note, does this work in IE 11? The MSDN page says that IE 11 requires the ms prefix

`var cryptoObj = window.crypto || window.msCrypto; // for IE 11`

File: src/components/ContentUploader/UploaderUtils.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class UploaderUtils {
+    static isMultiputSupported() {
```
Comment (wenboyu2) [inline src/components/ContentUploader/UploaderUtils.js]: can this be a function in `src/util/uploads.js`?

File: src/components/ContentUploader/UploaderUtils.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class UploaderUtils {
```
Comment (wenboyu2) [inline src/components/ContentUploader/UploaderUtils.js]: // @flow

File: src/components/ContentUploader/UploaderUtils.js (hunk 1 above)
Comment (DanDeMicco) [reply src/components/ContentUploader/UploaderUtils.js]: Didn't notice that file existed, oops! will move it there :)

Comment (wenboyu2) [reply src/components/ContentUploader/UploaderUtils.js]: or maybe `webcrypto.js`

**Comment by wenboyu2** [comment]
@JustinHoldstock that's a great point, we should also check for `msCrypto` and verify if you can still upload from IE 11.
`src/util/webcrypto.js L:13`

**Comment by DanDeMicco** [comment]
Good catch, verified ie11 works correctly after making the change

**Pull request merged**